reana-ui/src/components/NotificationBell.js

```diff
@@ -0,0 +1,61 @@
+/*
+  -*- coding: utf-8 -*-
+
+  This file is part of REANA.
+  Copyright (C) 2022 CERN.
+
+  REANA is free software; you can redistribute it and/or modify it
+  under the terms of the MIT License; see LICENSE file for more details.
+*/
+
+import { useSelector } from "react-redux";
+import { Popup, Icon, List } from "semantic-ui-react";
+
+import { getConfig, getUserQuota } from "~/selectors";
+import { getQuotaNotifications } from "~/pages/profile/components/Quota";
+import { healthMapping } from "~/util";
+
+export default function NotificationBell() {
+  const config = useSelector(getConfig);
+  const quota = useSelector(getUserQuota);
+
+  let notifications = getQuotaNotifications(quota, config.adminEmail);
+  const hasNotifications = !!notifications.length;
+  if (!hasNotifications) {
+    notifications.push({ body: "You have no notifications" });
+  }
+
+  return (
+    <Popup
+      trigger={
+        <Icon.Group size="large">
+          <Icon link name="bell outline" color="brown" />
+          {hasNotifications && (
+            <Icon corner="top right" name="circle" color="red" />
+          )}
+        </Icon.Group>
+      }
+      content={
+        <List divided relaxed>
+          {notifications.map((notification, index) => (
+            <List.Item key={index} as="a" href={notification.link}>
+              <List.Icon
+                verticalAlign="middle"
+                name={notification.icon}
+                color={healthMapping[notification.type]}
```

Comment:
It was a suggestion from @tiborsimko to make a small indication of how critical the notification messages are by coloring the icon. It currently looks like this:
![Screenshot from 2022-06-08 10-07-58](https://user-images.githubusercontent.com/4990025/172615377-73c55904-6199-4822-9e4e-625bf060c5d8.png)
![Screenshot from 2022-06-08 10-07-30](https://user-images.githubusercontent.com/4990025/172615416-103a92c4-084c-4a73-98bc-22a00e8a8ece.png)
If we would change it to display the grey color it would look like this:
![Screenshot from 2022-06-08 10-10-53](https://user-images.githubusercontent.com/4990025/172615555-748a37b7-4870-40db-a39e-88e3a8222b7b.png)

I don't know, both seem to be OK with me, maybe I'm already used to the colored ones. @tiborsimko , WDYT, should we switch back to the grey icons?

Reply:
Ok, switched to the grey color!